README.md

```diff
@@ -21,6 +21,9 @@ See the section about [running tests](https://facebook.github.io/create-react-ap
 
 ### `yarn test-cypress`
 
+**Prequisites:**
+All e2e-tests require an environment variable `CYPRESS_AUTH_QUERY_PARAMETERS`.
```

Comment:
Do we have a need for a package.json script for running cypress locally without the need of manually setting the environment variable for local tests?

Reply:
Can you elaborate? I do not understand the questions 🤔 😁

Reply:
"All e2e-tests require an environment variable `CYPRESS_AUTH_QUERY_PARAMETERS`." How to provide this environment variable if you wish to run cypress tests locally?